### PR TITLE
fix window lib path for renv; updates to work with new data flow

### PR DESCRIPTION
**Pull request by youyifong**
…ork with new data flow

### CODE REVIEW

File: cor_nonlinear/code/cor_nonlinear.R [hunk 1]
```diff
@@ -1,55 +1,70 @@
 #-----------------------------------------------
 # obligatory to append to the top of each script
+# There is a bug on Windows that prevents renv from working properly. saved.system.libPaths provides a workaround:
+if (.Platform$OS.type == "windows") saved.system.libPaths=paste0(Sys.getenv ("R_HOME"), "/library")
 renv::activate(project = here::here(".."))
+if (.Platform$OS.type == "windows") {
+    options(renv.config.install.transactional = FALSE)
+    renv::restore(library=saved.system.libPaths, prompt=FALSE) # for a quick test, add: packages="backports"
+    .libPaths(c(saved.system.libPaths, .libPaths()))
+} else renv::restore(prompt=FALSE)
```
Comment (benkeser) [inline cor_nonlinear/code/cor_nonlinear.R:10]: @nhejazi What's the recommendation for this? `restore` or `activate`?

I've been doing manual `restore()`s from time to time.

Comment (nhejazi) [reply cor_nonlinear/code/cor_nonlinear.R:10]: It would be better to avoid automatically calling `renv::restore()`

**Comment by nhejazi** [comment]
@youyifong --- would you mind creating a new PR that includes only the relevant changes to `cor_coxph` and `cor_nonpar`? I'm going to close this due to conflicts with the `make_dat_proc.R` script. If those changes are still necessary, please create a separate PR (that does not include any changes to `cor_coxph` and `cor_nonpar`) and re-submit.

**Comment by youyifong** [comment]
Hi Nima, in PR #1, I removed the lines related to restore in
make_dat_proc.R that I added.
Best,
Youyi

**Pull request closed without merging**